_component_design/tooltip.md

```diff
@@ -13,28 +13,304 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-Use the tooltip when you want to convey brief snippets of information, to
-explain an element a bit more, or to display truncated text.
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+{: .hxSubSectionTitle}
+- To provide in-context help information, ( e.g. when describing what will happen if you initiate a workflow)
+- To reveal hidden information ( e.g. truncated text, visualization data, or non-human-friendly data like unique ID alphanumerics)
+- Provide extra information to users, so that the screen isn’t overly full of unnecessary text.
+- An item on a page benefits from a supplemental description or further information.
+- Rarely used features or features that can be interpreted differently.
+- An item on a page does not have have a text label.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the tooltip pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-hero.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+### When not to use
+{: .hxSubSectionTitle}
+- If users must repeatedly refer to a tool tip to understand an option or to complete a workflow, then the information may need to be displayed on the page rather than hidden in a tooltip.
+- When users need to interact with the tip’s content.
+- When users work on the site on a mobile device. Tooltips do not appear on hover on a mobile device.
+
+### Best practice
+{: .hxSubSectionTitle}
+- Do not make the content obvious or just repeat what is already on the screen.
+- The tooltip is activated when hovered over  or on click.
+- Keep the tooltip text concise. Tooltips are perfect for short sentences and sentence fragments.
+- A tooltip disappears when mouse hovers away, or when clicked in the icon. 
+- Tooltip should be used on icon images, text, and table rows.
+- Tooltips should be placed near the object that is being hovered over. 
+- Tooltips should never be placed in a way that interferes with what the user is doing, or obscure the object of interest.
+- The maximum width for the tooltip is 400px.
+- Do not put interactive controls inside the tooltip.
+- Don't put images that look like they are interactive inside the tooltip.
```

Comment:
"Tooltips are perfect for short sentences and sentence fragments that assist the user in their understanding. The ideal tooltip is not obvious, nor does it repeat content that is already visible on the site. Place a tooltip in a subtle location, and ensure it does not interfere with the user or obscure any objects of interest. Use the tooltip on icon images, text, and table rows. The maximum width for the tooltip is 400px, ensure there are no interactive controls inside the tooltip."

Reply:
updated from UL to p with what you wrote.